com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -463,6 +488,148 @@ public override void OnInspectorGUI()
             m_SerializedCamera.serializedObject.ApplyModifiedProperties();
         }
 
+        private void UpdateStackCemerasToOverlay()
```

Comment:
question: I wonder if it makes sense to handle a `List<CameraRenderType> validCameraTypes`  when then, here we are assuming that the only valid type is `Overlay`